Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -377,9 +380,15 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
                 // destroy microwave
                 if (_tag.HasTag(item, "MicrowaveMachineUnsafe") || _tag.HasTag(item, "Metal"))
                 {
-                    component.Broken = true;
-                    SetAppearance(uid, MicrowaveVisualState.Broken, component);
-                    _audio.PlayPvs(component.ItemBreakSound, uid);
+                    if (_tag.HasTag(item, "Metal"))
+                    {
+                        _explosionSystem.QueueExplosion(uid, "Default", 10, 0.5f, 1, canCreateVacuum: false);
```

Comment:
you can just add the ExplosiveComponent to the microwave and then call Explode or w/e. That way you don't hardcode these values and other microwaves in the future can adjust them.

Reply:
> you can just add the ExplosiveComponent to the microwave and then call Explode or w/e. That way you don't hardcode these values and other microwaves in the future can adjust them.

Did I resolve it correctly?